CDP4Common/CDP4Common.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
     <Company>RHEA System S.A.</Company>
     <Title>CDP4Common Community Edition</Title>
-    <VersionPrefix>7.4.0</VersionPrefix>
+    <VersionPrefix>7.5.0</VersionPrefix>

```

Comment:
i have the impression that nothing has changed in the CDP4Common project, so why bump the version? if nothing has changed, please revert

Reply:
ah there is a change im Common that we wanted to include also that got forgotten, so thats coming, and then the version bumps to all the other projects (because of dependency) are justified.